root/types.js

```diff
@@ -1,20 +1,49 @@
-// @flow
-// Copyright (C) 2017 MetaBrainz Foundation
-//
-// This file is part of MusicBrainz, the open internet music database,
-// and is licensed under the GPL version 2, or (at your option) any
-// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+/*
+ * @flow
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
 
 
-// Types are in alphabetical order.
-//
-// The definitions in this file are intended to model the output of the
-// TO_JSON methods under lib/MusicBrainz/Server/Entity/; those are precisely
-// how data is serialized for us.
+/*
+ * Types are in alphabetical order.
+ *
+ * The definitions in this file are intended to model the output of the
+ * TO_JSON methods under lib/MusicBrainz/Server/Entity/; those are precisely
+ * how data is serialized for us.
+ */
+
+declare type AreaT = CoreEntityT & {
+  typeName: string;
```

Comment:
I realize the other object types mostly used semi-colons to separate the fields, but that's actually a [deprecated syntax](https://flow.org/en/docs/types/objects/) (they should use commas).